automata/fa/dfa.py

```diff
@@ -63,6 +61,10 @@ class DFA(fa.FA):
     _word_cache: List[DefaultDict[DFAStateT, List[str]]]
     _count_cache: List[DefaultDict[DFAStateT, int]]
 
+    ExpandStateFn = Callable[[DFAStateT], Iterable[Tuple[DFASymbolT, DFAStateT]]]
```

Comment:
I didn't know if there was a better place to add it

Reply:
@EduardoGoulart1 Decided to move these `*Fn` types to the top-level `dfa` module scope, since that is where the other DFA-specific types currently live.